config/environments/production.rb

```diff
@@ -106,4 +106,7 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  # Create a new production log file [daily, weekly, monthly,..]
+  config.logger = ActiveSupport::Logger.new(config.default_log_file, 'daily')
```

Comment:
**nitpick (blocking):** I wonder if we need to be more like the `RAILS_LOG_TO_STDOUT` block and only enable it when `RAILS_DAILY_LOG_ROTATION` env variable is set. In addition we probably need to make it more like the block above where we also set the log formatter.

Reply:
Good call. Updated in e9bf22dce2dd4d421140bc61a576a1d8c7aef510